tests/unit/lms/services/launch_verifier_test.py

```diff
@@ -206,4 +210,8 @@ def sign(pyramid_request):
     )
 
 
-pytestmark = pytest.mark.usefixtures("ai_getter")
+@pytest.fixture(autouse=True)
+def models(patch):
+    models = patch("lms.services.launch_verifier.models")
+    models.ApplicationInstance.get.return_value = mock.Mock(shared_secret="TEST_SECRET")
```

Comment:
I think it might be nicer to use a mock here for better test isolation. The `LaunchVerifier` service doesn't really depend on `models.ApplicationInstance` as such. It really just depends on the classmethod `models.ApplicationInstance.get()` and its contract, and then on the `.shared_secret` attribute of the returned `ApplicationInstance`. That said, I forgot to make it a `create_autospec()`.

A possible downside is that mocking might be slower here